tutorials/quick_start/main.py

```diff
@@ -98,7 +98,6 @@ def quantization_flow(config: Dict) -> Tuple[float, float, QuantInfo]:
     # Evaluate quantized model
     quant_results, dataset_info = ml.evaluate(quantized_model)
 
```

Comment:
I think you need to make a small change to the function find_modules in order to import the correct modules. don't you?